apps/desktop-wallet/src/pages/UnlockedWallet/UnlockedWalletLayout.tsx

```diff
@@ -153,24 +136,7 @@ const UnlockedWalletLayout = ({ children, title, className }: UnlockedWalletLayo
 
       <Scrollbar>
         <MainContent>{children}</MainContent>
-
-        <AppHeader title={title}>
-          {networkStatus === 'online' && (
-            <RefreshButton
-              role="secondary"
-              transparent
-              squared
-              short
-              onClick={refreshAddressesData}
-              disabled={isLoadingData}
-              aria-label={t('Refresh')}
-              data-tooltip-id="default"
-              data-tooltip-content={t('Refresh data')}
-            >
-              {isLoadingData ? <Spinner /> : <RefreshCw />}
-            </RefreshButton>
```

Comment:
We no longer need this refresh button, do we? Now we have polling.

Reply:
I guess we don't!